pgcli/pgexecute.py

```diff
@@ -186,7 +185,9 @@ def connect(self, database=None, user=None, password=None, host=None,
 
             cursor = conn.cursor()
 
-        superuser = self._select_one(cursor, "select current_setting('is_superuser')::bool")[0]
+        cursor.execute("SHOW ALL")
```

Comment:
Not a big fan assuming that super user is false for redshift. 

I think it makes sense to get all of the config vars if we decide to use it in the future. I'm satisfied. 👍